app/models/task.rb

```diff
@@ -0,0 +1,40 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :uuid             not null, primary key
+#  state       :integer
+#  name        :string
+#  description :text
+#  project_id  :uuid
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_tasks_on_project_id  (project_id)
+#
+
+class Task < ActiveRecord::Base
+  include Twil
```

Comment:
I personally wouldn't have done the text messaging as an AR callback as it means it always has to happen and makes things a little tricker for testing, etc.
I'd love to discuss alternative ways this could have been tackled when we discuss in person.
I do like the separation though as a module if you are going to do it this way.

Reply:
Thanks for the feedback @alexpeachey.... It would be fun to talk about.